tools/scripts/release-cleanup.ps1

```diff
@@ -0,0 +1,108 @@
+﻿# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#This script assumes that the OctoKit Nuget package is a custom install and is in the current folder.
+
+$deleteList = @()
+
+# Extracts a version from the release name.
+function Extract-Version($specificRelease){
+    $versionString = $specificRelease.Name
+    if($versionString -match ".*v(\d+\.\d+\.\d+\.\d+).*"){
+        return $matches[1]
+    } else {
+        $deleteList += $specificRelease
+        return [string]::Empty
+    }
+}
+
+# Sorts the releases based on the version number
+function Sort-Releases($releases){
+    $releaseMap = @{}
+    foreach($release in $releases.Result){
+        $releaseVersion = [System.version](Extract-Version $release)
+        if(-not [string]::IsNullOrEmpty($releaseVersion)){
+            $releaseMap.Add($releaseVersion,$release)
+        }
+    }
+
+    $releaseMap = $releaseMap.GetEnumerator() | Sort-Object -Descending -Property Name
+    return $releaseMap
+}
+
+# Delete a release
+function Delete-Release($release){
+    # Null checks
+    if([string]::IsNullOrEmpty($release.Id)){
+        Write-Host "No Release Id found"
+        return;
+    }
+
+    Write-Host "Deleting release" + $release.Name
+    $response = $instance.Repository.Release.Delete($repoId, $release.Id);
+
+    if(-not ($response.Result -eq "NoContent" -and $response.IsCompleted -eq $True)){
+        Write-Host "Failed to delete release -" $release.Name
+    } 
+}
+
+# Load the octokit dll
+Add-Type -Path ((Get-Location).Path + "\Octokit.0.32.0\lib\net45\Octokit.dll")
+
+# Get a new client with random product header value
+$productHeader = [Octokit.ProductHeaderValue]::new("AIWindows-CleanupScript")
+$instance = [Octokit.GitHubClient]::new($productHeader)
+
+# Add credentials for authentication
+$instance.Credentials = [Octokit.Credentials]::new("$(gitPATX)")
```

Comment:
>gitPATX [](start = 54, length = 7)

How is the actual PAT specified?

Reply:
The PAT needs to be a secret variable in the pipeline. I have documented the instructions [here](
![image](https://user-images.githubusercontent.com/45673569/55743285-c4c36280-59e6-11e9-9f07-cc7adf8ba4be.png)
).